src/autocomplete/index.ts

```diff
@@ -88,11 +91,15 @@ class Autocomplete {
     }
   }
 
-  updateProducts({ dataset: { query: selectedQuery, refinement, field } }: HTMLElement) {
+  updateProducts({ dataset: { query: selectedQuery, refinement, field, pastPurchase } }: HTMLElement) {
     const query = selectedQuery == null ? this.select(Selectors.autocompleteQuery) : selectedQuery;
     this.flux.emit('query:update', query);
-    // tslint:disable-next-line max-line-length
-    this.flux.saytProducts(field ? null : query, refinement ? [{ field: field || this.state.category, value: refinement }] : []);
+    if (pastPurchase !== undefined) {
```

Comment:
will this just be truthy when we want to make a `pastPurchase` request?

Reply:
I could check for truthiness but it was just to have it so passing in anything aside from undefined registers it as passed in.